maintainers/maintainer-list.nix

```diff
@@ -8155,6 +8155,12 @@
     githubId = 1829032;
     name = "Paul Hendry";
   };
+  polygon = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
done